assets/js/googlesitekit/datastore/user/surveys.js

```diff
@@ -168,6 +170,40 @@ const baseSelectors = {
 		return state.currentSurveySession;
 	},
 
+	/**
+	 * Gets the completion triggers for the current survey, if one exists.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return {Array|null|undefined} Current survey's completion triggers if available; undefined if loading and `null` if no questions/survey are found.
+	 */
+	getCurrentSurveyCompletions: createRegistrySelector( ( select ) => () => {
+		const currentSurvey = select( CORE_USER ).getCurrentSurvey();
+
+		if ( currentSurvey === undefined ) {
+			return undefined;
+		}
```

Comment:
`getCurrentSurvey` is a little different than other selectors. It doesn't have a resolver attached to it and so the initial state is `null`, so the `undefined` condition would likely never be true here and can be removed.